app/templates/sparql.hbs

```diff
@@ -0,0 +1,10 @@
+{{page-title "Sparql"}}
+<div class="au-c-body-container au-c-body-container--scroll">
```

Comment:
Optional but might be nice. You could wrap all this in a `StaticPage` component which will add the header and footer like on the other "static pages". Might look nicer?